packages/url_launcher/url_launcher_ios/ios/url_launcher_ios/Sources/url_launcher_ios/URLLaunchSession.swift

```diff
@@ -12,6 +12,7 @@ final class URLLaunchSession: NSObject, SFSafariViewControllerDelegate {
 
   private let completion: OpenInSafariCompletionHandler
   private let url: URL
+  private var isLoadCompleted: Bool?
```

Comment:
A nullable boolean should only be used when there are three conceptually different states. The code in this PR only actually recognizes two.